example/spec/parallel/SyncGranuleTask/TestAlternateFileBucketSpec.js

```diff
@@ -0,0 +1,155 @@
+
+'use strict';
```

Comment:
This is a component integration test that runs sync granule with a payload like what we're expecting to work for this ticket.